src/components/forms/inputs/SelectInput.jsx

```diff
@@ -31,10 +31,12 @@ export default class SelectInput extends InputBase {
         }
 
         return (
-            <select value={ this.props.value || '0' }
-                onChange={ this.onChange.bind(this) }>
-                { optionElements }
-            </select>
+            <div className="SelectInput">
```

Comment:
React-beginner: Just learned about the  `className` property!

https://reactjs.org/docs/faq-styling.html